### ISSUE TITLE
Corpus format

### ISSUE THREAD

**Issue by yuningkang** [lyeoni/gpt-pytorch#1]
Hi sir，Thanks for your implementation！Could you please show me a simple example of the training corpus format

**Comment by lyeoni** [comment]
- For pre-training, a sentence for a line
```
# format: [SENTENCE]
Hello, I am lyeoni.
I'm working in NAVER.
```
- For fine-tuning(classification), label and sentence for a line.
```
# format: [CLASS_LABEL]\t[SENTENCE]
POSITIVE\tIt's so interesting.
NEGATIVE\tI don't like this movie.
```

For more information, please refer `data_utils.py`
```
        corpus_path = args.train_corpus if mode=='train' else args.test_corpus
        with open(corpus_path, 'r', encoding='utf-8') as reader:
            corpus = reader.readlines()
        
        # Create examples
        if args.pretrain:
            corpus = list(map(lambda x: x.strip(), corpus))
            corpus = list(filter(lambda x: len(x) > 0, corpus))
            examples = [PretrainInputExample(text) for text in corpus]
        elif args.finetune:
            corpus = list(map(lambda x: x.split('\t'), corpus))
            corpus = list(map(lambda x: list(map(lambda y: y.strip(), x)), corpus))
            corpus = list(map(lambda x: list(filter(lambda y: len(y) > 0, x)), corpus))
            examples = [ClsInputExample(text, label) for label, text in corpus]
```
